### PR TITLE
feat(i18n,curriculum): add Bilibili ids for Chinese

### PR DESCRIPTION
**Pull request by miyaliu666**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

**Review by Fhfdghbf — DISMISSED** [review]

**Comment by miyaliu666** [comment]
I've added bilibili ids to all videos in four Python certifications. The first two commits are respectively for the first certifications. Please let me know if there's any issue. Thanks! @ojeytonwilliams

**Review by ojeytonwilliams — APPROVED** [review]
It all looks good to me, @miyaliu666!

I rebased the PR to include the changes in https://github.com/freeCodeCamp/freeCodeCamp/pull/42869 so, hopefully, the tests will pass now.

**Comment by ojeytonwilliams** [comment]
I'm not sure why the tests weren't running, but I closed and re-opened the PR and it seems to have got them going again.

File: curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/iterations-more-patterns.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ id: 5e7b9f070b6c005b0e76f05f
 title: 'Iterations: More Patterns'
 challengeType: 11
 videoId: 9Wtqo6vha1M
+bilibiliIds:
+  aid: 334491369
+  bvid: BV1tw411R7Mm
+  cid: 376530765
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/iterations-more-patterns.md:9]: These are the ids for https://www.freecodecamp.org/learn/scientific-computing-with-python/python-for-everybody/iterations-loop-idioms do you have the correct ones?

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
They all work well, with the one exception I mentioned.

File: curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/iterations-more-patterns.md (hunk 1 above)
Comment (miyaliu666) [reply curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/iterations-more-patterns.md:9]: [Here](https://www.bilibili.com/video/BV1tw411R7Mm?spm_id_from=333.999.0.0) is the video. The ids are correct. I don't what's wrong with it.

`<iframe src="//player.bilibili.com/player.html?aid=334491369&bvid=BV1tw411R7Mm&cid=376530765&page=1" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>`

Comment (scissorsneedfoodtoo) [inline curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/iterations-more-patterns.md:9]: These seem to be the right values for the "05 - Iterations D - Python for Everybody Course" video used in this challenge:

```suggestion
  aid: 674492981
  bvid: BV1hU4y1H7tF
  cid: 376531204
```

Here's the `iframe` for quick testing:

```
<iframe src="//player.bilibili.com/player.html?aid=674492981&bvid=BV1hU4y1H7tF&cid=376531204&page=1" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>
```

Comment (miyaliu666) [reply curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/iterations-more-patterns.md:9]: Got it. Thank Kris for finding the correct ones.

Comment (ojeytonwilliams) [reply curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/iterations-more-patterns.md:9]: Perfect, thanks @miyaliu666 and @scissorsneedfoodtoo!

**Review by ojeytonwilliams — APPROVED** [review]

**Review by raisedadead — APPROVED** [review]

**Pull request merged**